PatientRegistrationApp/src/App.vue

```diff
@@ -432,6 +448,10 @@ export default {
 </script>
 <style lang="css" scoped>
 
+	.v-btn:not(.v-btn--round).v-size--default {
+		font-size: 1.2rem;
+		min-width: 200px;
```

Comment:
Can we stay consistent with rem vs. px? I'd say stick with rem since px can change based on your monitor...

Reply:
Yep!  Changed 200px to 12rem